.github/workflows/pr-check.yaml

```diff
@@ -76,6 +79,9 @@ jobs:
       - name: Execute pnpm
         run: pnpm install
 
+      - name: Execute Build
```

Comment:
Unrelated should be another PR

Reply:
The PR will fails otherwise, the pr-check must build the packages otherwise typecheck will not able to infer the types. The backend package depends on the podlet-js package.

I can do it in a separate PR, but this one will need to rebase.

Reply:
@jeffmaury  the PR has been rebased :+1: